code/modules/events/wizard/greentext.dm

```diff
@@ -87,7 +87,7 @@
 		var/message = "<span class='warning'>A dark temptation has passed from this world"
 		if(M in color_altered_mobs)
 			message += " and you're finally able to forgive yourself"
-			M.color = initial(M.color)
+			M.remove_atom_colour(ADMIN_COLOUR_PRIORITY)
```

Comment:
This should make sure the admin color is one of the two colors this can cause, just in case something else gets them/an admin fucks around